openghg/retrieve/met/_ecmwf.py

```diff
@@ -145,13 +237,13 @@ def pull_met(
 
             dataset_savepath = os.path.join(
```

Comment:
There's quite a nice feature in Python now where if you use the `pathlib` module (`from pathlib import Path` ) you can use a `Path` object rather than having to use the `os.path.join` etc. sytnax you can write this in a neater way where it knows it's a path:

```
dataset_savepath = Path(save_path) / f"Met_{site}_{network}_{year}{month}.nc"
```

This is generally the kind of syntax we've been using when accessing or creating paths to files.